src/mango/src/mango_cursor_view.erl

```diff
@@ -252,6 +252,11 @@ execute(#cursor{db = Db, index = Idx, execution_stats = Stats} = Cursor0, UserFu
                         UserFun, Cursor, Stats1, FinalUserAcc0
                     ),
                     {ok, FinalUserAcc1};
+                {ok, Error} when is_tuple(Error) ->
+                    % fabric_view_all_docs turns {error, Resp} results into {ok, Resp}
+                    % for some reason. If we didn't get a proper cursor record, assume
```

Comment:
should we not fix that instead of papering over the odd behaviour? (he said without looking how hard that would be :)

Reply:
It’s been like that for years and it’s probably expected to return that in other places where all_docs is used. Since we don’t have rigorous tests for error conditions we’d probably break something else.